libs/spark/src/NavBarButton.tsx

```diff
@@ -3,16 +3,22 @@ import { withStyles } from '@material-ui/core/styles';
 import { Button, ButtonProps } from '@material-ui/core';
 import { theme } from './styles';
 
-export type NavBarButtonProps = Omit<ButtonProps, 'variant' | 'color' | 'size'>;
+export type NavBarButtonProps = Omit<ButtonProps, 'variant' | 'color'>;
 
 export const NavBarButton = withStyles({
   root: {
-    height: 48,
     borderRadius: 8,
+    backgroundColor: 'transparent',
+    border: '2px solid transparent',
     padding: 8,
     textTransform: 'none',
     color: theme.palette.text.onLightLowContrast,
-    fontSize: '1.125rem',
+    transition: theme.transitions.create(
+      ['background-color', 'box-shadow', 'color'],
+      {
+        duration: theme.transitions.duration.short,
+      }
+    ),
```

Comment:
it looks like this isn't necessary after all. I tried commenting it out and it no longer lags, so it was due to the progressive layout shifts solved by a constant border size.
```suggestion
```